### DISCUSSION TITLE
Is there a clear documentation on selfhosting and tftp ?

### DISCUSSION THREAD

**Discussion by Denyreal** [antonym/netboot.xyz#1427] [Q&A]
Hi friends .Hope you're doing well.

Basically netboot.xyz is a terrific project. I'm trying it in a lab and have plans to have this in productions but there are two caveats here. 

1. the selfhosting "documentation" is not a documentation and not a clear how to neither. I can't find explicitely where we deals with the tftp server as it's the very basis of any PXE infrastructure. In the ansible role vars i can't find a lcea rstatement which declare what is the tftp server option i need. 

2. Far from me the idea to think of some nasty plans at all but to use is in production we should assume all vmlinuz/initrd are legit right ? Signatures checked and files downloaded from official repos right ? Is there any clear statement on why signatures seems to be disabled by default and why some vmlinuz seems to be downloaded from this github instead of officials distro hosting .

I' am this annoying person but this project looks terrific and unfortunately seems to have some "black magic " vibes . Documentation is the key.  i know it 's always hard to find contributors to document things but it would be great to have a contributing effort on this matter.

Cheers and thanks to anyone who can enlight me ;)

**Comment by diaverso** [comment]
See if this helps.
https://netboot.xyz/docs/docker/

The problem I have is that I have the dhcp through pi-hole and not in docker.
I created this post to see if someone can help me
https://github.com/orgs/netbootxyz/discussions/1430

**Reply by diaverso** [reply]
I found these 2 more informations, in case they are helpful
https://netboot.xyz/docs/booting/ipxe
https://netboot.xyz/docs/booting/tftp
